esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -811,6 +811,23 @@ bool CwssqlEx::onGetRelatedIndexes(IEspContext &context, IEspGetRelatedIndexesRe
     return true;
 }
 
+void CwssqlEx::processMultipleClusterOption(StringArray & clusters, const char  * targetcluster, StringBuffer & hashoptions)
+{
+	int clusterscount = clusters.length();
+	if (clusterscount > 0)
+	{
+		hashoptions.appendf("\n#OPTION('AllowedClusters', '%s,", targetcluster);
+		ForEachItemIn(i,clusters)
+		{
+			if (!isValidCluster(clusters.item(i)))
+				throw MakeStringException(-1, "Invalid alternate cluster name: %s", clusters.item(i));
+
+			hashoptions.appendf("%s%s", clusters.item(i), i + 1 == clusterscount ? "" : ",");
```

Comment:
Minor: if you does not append the "," in line 819, you may use hashoptions.appendf(",%s", clusters.item(i)) here.